src/consensus/aft/raft.h

```diff
@@ -380,6 +414,11 @@ namespace aft
       return state->commit_idx;
     }
 
+    Index get_commit_idx_unsafe()
```

Comment:
Can you please change the unsafe version to just aquire a lock and then call the unsafe function

Reply:
Not sure what you mean. The safe version gets the lock, the unsafe version is not supposed to get the lock?